packages/worker-capacity-manager/src/index.ts

```diff
@@ -33,7 +35,11 @@ const updateASGCapacity = async (
 		throw new Error("absoluteMinCapacity can't be greater than asgMaxCapacity");
 	}
 
-	const minCapacity = Math.min(totalMessagesInQueue, asgMaxCapacity);
+	// It takes so long to start a new instance, we assume that it will generally
+	// be faster to allocate multiple jobs to the same box
+	const numServersForMessages = totalMessagesInQueue % JOBS_PER_BOX;
```

Comment:
modulus gives you the remainder in a division
```suggestion
	const numServersForMessages = Math.ceil(totalMessagesInQueue / JOBS_PER_BOX);
```